### PR TITLE
[Prefix]Add visual indicator for Reserved prefix Packages

### PR DESCRIPTION
**Pull request by shishirx34**
Fixes: #4383 #4384 #4378 
Also, updated `LuceneSearchService` to index `IsVerified` for local testing of reserved namespaces in gallery.

Screenshots:
Details page:
image

Search Page:
image

Profile Page:
image

### CODE REVIEW

File: src/NuGetGallery/Views/Shared/_ListPackage.cshtml [hunk 1]
```diff
@@ -12,6 +12,14 @@
             <div class="package-header">
                 <a class="package-title" href="@Url.Package(Model.Id, Model.UseVersion ? Model.Version : null)">@Html.BreakWord(Model.Id)</a>
 
+                @if (Model.IsVerified.HasValue && Model.IsVerified.Value)
+                {
+                    <img style="vertical-align: top; margin-left: 3px; margin-top: 3px; width: 20px"
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Shared/_ListPackage.cshtml]: >style [](start = 25, length = 5)

Move this to CSS

File: src/NuGetGallery/Views/Shared/_ListPackage.cshtml [hunk 2]
```diff
@@ -12,6 +12,14 @@
             <div class="package-header">
                 <a class="package-title" href="@Url.Package(Model.Id, Model.UseVersion ? Model.Version : null)">@Html.BreakWord(Model.Id)</a>
 
+                @if (Model.IsVerified.HasValue && Model.IsVerified.Value)
+                {
+                    <img style="vertical-align: top; margin-left: 3px; margin-top: 3px; width: 20px" 
+                         src="~/Content/gallery/img/reserved-indicator.svg" 
+                         @ViewHelpers.ImageFallback(Url.Absolute("~/Content/gallery/img/reserved-indicator-20x20.png"))
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Shared/_ListPackage.cshtml]: >ViewHelpers [](start = 26, length = 11)

The fallback seems unnecessary as the `reserved-indicator.svg` is static content. Thoughts?

Comment (shishirx34) [reply src/NuGetGallery/Views/Shared/_ListPackage.cshtml]: fallback is for browsers that don't render svg, some older IE versions etc. This is common through out our codebase.

File: src/NuGetGallery/Views/Shared/_ListPackage.cshtml [hunk 3]
```diff
@@ -12,6 +12,14 @@
             <div class="package-header">
                 <a class="package-title" href="@Url.Package(Model.Id, Model.UseVersion ? Model.Version : null)">@Html.BreakWord(Model.Id)</a>
 
+                @if (Model.IsVerified.HasValue && Model.IsVerified.Value)
+                {
+                    <img style="vertical-align: top; margin-left: 3px; margin-top: 3px; width: 20px" 
+                         src="~/Content/gallery/img/reserved-indicator.svg" 
+                         @ViewHelpers.ImageFallback(Url.Absolute("~/Content/gallery/img/reserved-indicator-20x20.png"))
+                         title="@Strings.ReservedNamespace_ReservedIndicatorTooltip"/>
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Shared/_ListPackage.cshtml]: >ReservedNamespace_ReservedIndicatorTooltip [](start = 41, length = 42)

I don't think we use Strings anywhere else in views. Maybe use just a plain C# string?

Comment (shishirx34) [reply src/NuGetGallery/Views/Shared/_ListPackage.cshtml]: I think using the strings is most appropriate. This needs to be reused in another place as well.

**Pull request merged**